csse_covid_19_data/README.md

```diff
@@ -117,7 +117,7 @@ We are also monitoring the curve change. Any errors made by us will be corrected
 * June 13, Through data provided by the Michigan Department of Health and Human Service’s (MDHHS) Communicable Disease Division, we were able to appropriately distribute the probable cases MDHHS began reporting on June 5th.
 * June 12, Louis City, MO, data for confirmed cases and deaths from March 16 to June 11 were updated to match up with the updated official report at the [City of St. Louis dashboard](https://www.stlouis-mo.gov/covid-19/data/index.cfm). Date of the first case was updated to March 16, and date of the first deaths was updated to March 23.
 * June 12, Louis County, MO, data for confirmed cases and deaths from March 9 to June 11 were updated to match up with the updated official report at [St. Louis County government site](https://stlcorona.com/resources/covid-19-statistics1/). Date of the first case was remained on March 8, and date of the first deaths was updated to March 20.
-* June 12, Massachusetts, cases from April 15 to June 11 were updated to match official updateded statistics from the [Massachusetts government raw data - County.csv](https://www.mass.gov/info-details/covid-19-response-reporting). This change arose due to to release of historical probable cases by the state. The alteration distributes probable cases and updates some confirmed case counts that were revised by the state. Dukes and Nantucket are still reported together, though County.csv lists them separately.

```

Comment:
Have you checked that this type doesn't occur more than one? i.e from copy/paste

Reply:
I read the whole of document and I find this typo, so, I did the job: fixed and did a pull request.